src/twisted/web/websocket.py

```diff
@@ -0,0 +1,32 @@
+# -*- test-case-name: twisted.web.test.test_websocket -*-
+"""
+Websocket (rfc6455) client and server support.
+
+For websocket servers, place a L{WebSocketResource} into your Twisted Web
+resource hierarchy.
+
+For websocket clients, call L{WebSocketClientEndpoint.connect}.
```

Comment:
Minor comment

```suggestion
For websocket clients, create a new endpoint via L{WebSocketClientEndpoint.new} with the WebSocket server URL
and then, on the newly created endpoint, call L{WebSocketClientEndpoint.connect}.
```

    endpoint = WebSocketClientEndpoint.new(reactor, "ws://localhost:8080/webskt.rpy")
    print("connecting...")
    await endpoint.connect(MyClient())